extensions/map-layers-formats/src/ArcGisFeature/ArcGisFeatureProvider.ts

```diff
@@ -399,7 +399,7 @@ export class ArcGisFeatureProvider extends ArcGISImageryProvider {
 
     const doFeatureInfoQuery = async (format: ArcGisFeatureFormat, outFields?: string, returnGeometry?: boolean) => {
       const infoUrl = this.constructFeatureUrl(quadId.row, quadId.column, quadId.level, format, "standard", queryEnvelope,
-        outFields, undefined, returnGeometry, toleranceWorld);
+        outFields, undefined, returnGeometry, tilePixelSize);
```

Comment:
Why was this changed from `toleranceWorld`?

Reply:
That's a good question...

There are two concepts:
1. **Query tolerance**: which is used to increase the area around a given point we want to query.
2. **maxAllowedOffset**: which is used for generalizing geometries returned by the query operation.

Before those changes, we were using the `tolerance` value to specify the `maxAllowedOffset` value, which was wrong.  We want the tolerance to be set around 7 pixels, and maxAllowedOffet to be around 1 pixel (bigger value means coarser geometries will be returned by the server).
Pixels values are converted in world coordinates before sending it to the ArcGIS service.